src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="header">
+    <div class="header-content container">
+        <div class="header-logo-label">
+            <div class="header-logo"><img alt={{img_text}} src="../../../assets/icons/logo.svg"></div>
+            <div class="header-label">Todos</div>
+        </div>
+        <div class="header-search">
+            <button><img alt="My image name" src="../../../assets/icons/searchIcon.svg"></button>
```

Comment:
ditto

Reply:
i have introduced a class property named imageUrl for the purpose vaia.